src/main/java/com/codeborne/selenide/SelenideDriver.java

```diff
@@ -53,9 +53,9 @@ public SelenideDriver(Config config, Driver driver) {
     this.driver = driver;
   }
 
-  public SelenideDriver(Config config, WebDriver webDriver, @Nullable SelenideProxyServer selenideProxy) {
+  public SelenideDriver(Config config, WebDriver webDriver, @Nullable SelenideProxyServer selenideProxy, File browserDownloadsFolder) {
```

Comment:
Should we have a constructor with the default browser downloads folder? 

I mean what if we brake the usage of `public SelenideDriver(Config config, WebDriver webDriver, @Nullable SelenideProxyServer selenideProxy)` constructor?

Reply:
Right. I will restore the previous constructor too.

Reply:
done in commit b23338445e7